frontend/src/Context/PreviewPublisherProvider/usePreviewPublisher/usePreviewPublisher.tsx

```diff
@@ -170,7 +170,6 @@ const usePreviewPublisher = (): PreviewPublisherContextType => {
 
       try {
         const permissionStatus = await window.navigator.permissions.query({
-          // @ts-expect-error The camera and microphone permissions are supported on all major browsers.
```

Comment:
[`camera`](https://caniuse.com/mdn-api_permissions_permission_camera) and [`microphone`](https://caniuse.com/mdn-api_permissions_permission_microphone) are now supported in FF, the last holdout.